ui-admin/src/study/surveys/SurveyEditorView.tsx

```diff
@@ -12,6 +13,25 @@ type SurveyEditorViewProps = {
   onSave: (update: { content: string }) => Promise<void>
 }
 
+type Draft = {
+  content: string
+  date: number
```

Comment:
`date` is currently unused. I want to use this info in the un-finished modal to tell the user when the draft is from so they can get some sort of basic context